Code/Framework/AzToolsFramework/AzToolsFramework/API/ToolsApplicationAPI.h

```diff
@@ -124,6 +124,8 @@ namespace AzToolsFramework
 
         /*!
          * Fired just after applying a requested undo or redo operation.
+         * Note that prefab propagation will not have occurred at this point, so data may not yet be updated.
+         * Consider listening to OnPrefabInstancePropagationEnd on PrefabPublicNotificationBus instead.
```

Comment:
Thanks for adding this comment! I'm still in favour of seeing this removed in future as it can lead to these sorts of bugs 😬

Reply:
Excellent! Thanks for doing that ❤️